src/features/zaaksysteem/service.ts

```diff
@@ -130,12 +131,10 @@ export const useZakenByKlantBedrijfIdentifier = (
         "rol__betrokkeneIdentificatie__vestiging__vestigingsNummer",
         searchParam.vestigingsnummer,
       );
-    } else if ("nietNatuurlijkPersoonIdentifier" in searchParam) {
-      url.searchParams.set(
-        "rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__innNnpId",
-        searchParam.nietNatuurlijkPersoonIdentifier,
-      );
-    }
+    } else if ("rsin" in searchParam) {
```

Comment:
ik denk dat een comment hier wel nuttig is.  aangezien we afwijken van het pattern waarbij we tussen front-end en backend zoveel mogelijk de api's volgen.
dus iets in de trant:  In de daadwerkelijke call naar de zaaksystemen  wordt rsin of kvknummer in het veld "rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__innNnpId" gestopt. Aangezien we voor een klant tegelijkertijd verschillende zaaksystemen willen kunnen bevragen, waarbij de een rsin en de ander kvknummer hanteert, zullen we beiden mee moeten geven en in de backend per zaaksysteem de rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__innNnpId toevoegen met de juiste waarde